api/services/userService.js

```diff
@@ -0,0 +1,37 @@
+const jwt = require("jsonwebtoken");
+const axios = require("axios");
+
+const { userDao } = require("../models");
+
+const kakaoSignUp = async (kakaoToken) => {
+  const getKakaoUserData = await axios.get(
+    "https://kapi.kakao.com/v2/user/me",
+    {
+      headers: {
+        Authorization: `Bearer ${kakaoToken}`,
+      },
+      "Content-type": "application/x-www-form-urlencoded;charset=utf-8",
+    }
+  );
+
+  const kakaoId = getKakaoUserData.data.id;
+  const profileNickname = getKakaoUserData.data.properties.nickname;
+  const accountEmail = getKakaoUserData.data.kakao_account.email;
+  const ageRange = getKakaoUserData.data.kakao_account.age_range;
```

Comment:
```suggestion
  const { id: kakaoId, properties: { nickname: profileNickname }, kakao_account: { email: accountEmail, age_range: ageRange } } = getKakaoUserData.data;
```
구조 분해 할당을 활용할 수 있도록 해주세요.